### ISSUE TITLE
Curry function of composeAll needs an input.

### ISSUE THREAD

**Issue by ndarilek** [kadirahq/react-komposer#42]
Getting this error a *lot* using react-komposer 1.3.1 as shipped with mantra-core.

I acknowledge that this isn't a very helpful bug report as I can't create an easy reproduction. FWIW, the code that triggers it is:

```
import {useDeps} from "mantra-core"

import ResetPassword from "../components/reset_password.jsx"

console.log("ResetPassword", useDeps()(ResetPassword))

export default useDeps()(ResetPassword)
```

And I can't pin down what's causing it. If I change ResetPassword to a dirt simple `<p>It worked</p>` component, I still get the error. If I switch back to a more verbose container that lumped the Mantra action in with the container and uses composeAll/a composer function, it works. I'd blame the container, but I'm literally using the very same code in several others (with component names changed, of course) and it works fine. I'm guessing somewhere in the render chain is getting an undefined component, but this error message is not very helpful.

Can it be made more so, giving a component name, the value it actually received, etc.?

Also, is there a chance an error is being swallowed somewhere? In the past when I've hit this, I've found errors in child components that should have been reported but weren't. Given that, I'm wondering if something in the component composition isn't reporting errors correctly. If I remove the container and render the component directly, everything is fine (except for the fact that it isn't getting its data, of course.)

Thanks.

**Comment by arunoda** [comment]
This error comes when you are using `composeAll` and not passing a component.
Here's the related test: https://github.com/kadirahq/react-komposer/blob/5d2d47859384c3cc112f2672ecad1dbb5ab72cca/lib/__tests__/compose_all.js#L30

**Comment by ndarilek** [comment]
Right, I just wonder if it might be made more helpful. Maybe you could 
iterate through the list of passed components and display their names so 
I know which composeAll is failing?

I'm dealing with [this 
issue](https://github.com/meteor/meteor/issues/6364) which looks like 
either a Meteor module failure or something too subtle for me to spot. 
But, either way, the code I'm changing is not a composeAll call and this 
is the exception I'm getting, so it would be useful if it could help me 
to pinpoint which composeAll call is failing.

**Comment by ndarilek** [comment]
So I have a minimal reproduction of an app that demonstrates this error. It uses mantra-core, and the issue seems to go away when I eliminate a container and render my UI components directly. If I comment out an import for an unused module, my app works. With the import, it crashes with the error in this issue title, and the entire module containing the import fails to load (thus resulting in the UI component being undefined and, in turn, this error.)

I don't know if it's an issue with mantra-core, react-komposer, Meteor 1.3 modules or my own code. If the issue was in my own code, I'd expect to get better error reports in my console logs. Since I don't know where the issue lies, I'm posting it here too, because I don't see it if I just ditch containers and render my UI components directly.

[Here is the reproduction](https://github.com/ndarilek/meteor-modules-mystery), and [here is the related Meteor issue](https://github.com/meteor/meteor/issues/6364).

Thanks.
